unpackaged/config/rd2_post_config/objects/npe03__Recurring_Donation__c.object

```diff
@@ -282,6 +282,20 @@
         <trackTrending>false</trackTrending>
         <type>Picklist</type>
     </fields>
+    <fields>
+        <fullName>%%%NAMESPACE%%%InstallmentFrequency__c</fullName>
```

Comment:
Thanks for adding the namespace here @sam-knox. One more change needed. This field has to be added to the `unpackaged/config/rd2_post_config/package.xml` file because it hasn't been included in the unmanaged metadata prior to this change to the helptext. For example: https://github.com/SalesforceFoundation/NPSP/pull/5134/files#diff-d903a33300214cbaa2763d0d2d73ace4

Reply:
@force2b - I just added InstallmentFrequency to the package.xml file. Let me know if there's anything else needed here. Sorry I missed your change request from yesterday!